packages/site-admin/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## 0.0.1-beta.1
```

Comment:
Are we going to actually release this?

If not, it might make sense to phrase this as "Next" or "Unreleased".

Reply:
I believe we should. Otherwise, how can we export these resources from this monorepo?

Reply:
I see. Yes, if we'll be using this outside of the monorepo to ship something, it will need to be published. Somehow I assumed we're not there yet as this looks in an early phase.

Reply:
We should publish this package ASAP to address requirements from other projects/folks.